_posts/2017-01-17-Oredev2016.md

```diff
@@ -0,0 +1,696 @@
+When designing a mobile form, be sure you do it on a mobile phone.
+
+<p style="text-align: center;">
+  <img style="max-width:640px;" alt="Oredev begin" src="/img/oredev/ux/physicalchallenge.png">
+</p>
+
+##### ***Invisible element on phone***
+Accidental activation if shutdown
+
+<p style="text-align: center;">
+  <img style="max-width:640px;" alt="Oredev begin" src="/img/oredev/ux/accidentalInvisible.png">
+</p>
+
+### Responsive Traps
+* Be sure your response is half a second and not greater.
+* If the response is greater show a loading circle or show the log of the process
+
+### Efficiency Traps
+##### ***Information overload***
+Have a page that shows you too much information, be sure to keep it as simple as possible.
+##### ***System amnesia***
+When they show you information that you don't need
+
+<p style="text-align: center;">
+  <img style="max-width:640px;" alt="Oredev begin" src="/img/oredev/ux/systemamnesia.png">
+</p>
+
+##### ***Autocorrect***
+The name says it all, when autocorrect changed your word in something you don't need.
+
+
+
+### Habituating Traps
+* Doing things so many times it gets automatic and then it changes.
+
+##### ***Non redundant gratuitous redundancy***
+Too much of the same action (ex. if you have one button that does multiple different actions on other systems)
+
+<p style="text-align: center;">
+  <img style="max-width:640px;" alt="Oredev begin" src="/img/oredev/ux/redudancy.png">
+</p>
+
+##### ***Inconsistency***
+Message button on an older version is on another place or it is replaced with text
+
+<p style="text-align: center;">
+  <img style="max-width:640px;" alt="Oredev begin" src="/img/oredev/ux/inconsistency.png">
+</p>
+
+### Beautiful Traps
+##### ***Unattractive design***
+Not appealing design where the user will not stay on the website.
+
+
+
+### Best Practices
+* Identify common scenarios using the site
+* Walk through all the ways the user could complete each task
+* Identify and log any traps you observe
+* If you can't find a trap, identify the tenet
+* Document and discuss
+
+
+<a name="api" />
+
+## Feed the links, tuples' a bag – An introduction to Linked Data APIs (Sebastien Lambla)
+
+<img class="float-image p-image" src="{{ '/img/oredev/sebastien.jpg' | prepend: site.baseurl }}" alt="Sebastien Lambla" style="width: 100px"/>
+
+[Sebastien Lambla](http://www.twitter.com/serialseb) is a keen open-source advocate and contributor, a long-time REST proponent
+and distributed systems aficionado With a career spanning over 20 years on many platforms,
+he's a keen speaker and trainer, and has been known to talk a lot about technical things and unicorns.
+
+Started with a calm song of tuples' a bag from Mary Poppins Sebastien guides us through a possible future of HTTP API's.
+When talking about creating our own protocol you know that it is recommend not to invent the third wheel and use HTTP instead.
+Someone else did the hard work, so why should you?
```

Comment:
Someone else already did all the hard work, so why should you do it all over again?